project.yml

```diff
@@ -0,0 +1,33 @@
+name: CombineX-Carthage
+options:
+  deploymentTarget:
+    iOS: 8.0
+    macOS: !!str '10.10'
+    tvOS: 9.0
+    watchOS: 2.0
+targets:
+  CXLibc:
+    templates: [Framework]
+  CXUtility:
+    templates: [Framework]
+  CXNamespace:
+    templates: [Framework]
+  CombineX:
+    templates: [Framework]
+    dependencies:
+      - target: CXLibc_${platform}
+      - target: CXUtility_${platform}
+      - target: CXNamespace_${platform}
+  CXFoundation:
+    templates: [Framework]
+    dependencies:
+      - target: CXUtility_${platform}
+      - target: CXNamespace_${platform}
+      - target: CombineX_${platform}
+targetTemplates:
+  Framework:
```

Comment:
Better setting `APPLICATION_EXTENSION_API_ONLY` to `YES`.

Reply:
Added!